petastorm/pytorch.py

```diff
@@ -150,6 +148,23 @@ def __iter__(self):
 
         keys = None
 
+        if self._in_iter:
+            raise RuntimeError(_PARALLEL_ITER_ERROR)
+        if self._in_iter is not None:
+            self.reader.reset()
+            logging.warn('Start a new pass of Petastorm DataLoader, reset underlying Petastorm reader to position 0.')
```

Comment:
Let's use 
```python
logger = logging.getLogger(__name__)
logger.warn(..)
```
this will give our users better granularity in the control over log levels of different parts of their system.